### PR TITLE
added the nginx redirected logic

### PR DESCRIPTION
**Pull request by arbabnazar**
@edx/devops kindly review. Thanks

### CODE REVIEW

**Comment by e0d** [comment]
@feanil can you review?

**Comment by arbabnazar** [comment]
@feanil here is a change for your reference: https://github.com/edx/configuration/pull/2682

File: playbooks/roles/ansible-role-django-ida/templates/templates/edx/app/nginx/sites-available/ROLE_NAME.j2 [hunk 1]
```diff
@@ -57,14 +57,30 @@ server {
...
+  # there is an unfortunate mix of conditonals here.
+  {{ '{%' }} if NGINX_REDIRECT_TO_HTTPS {{ '%}' }}
+     {{ '{%' }} if NGINX_HTTPS_REDIRECT_STRATEGY == "scheme" {{ '%}' }}
+  # Redirect http to https over single instance
+  if ($scheme != "https") 
+  { 
+   set $do_redirect_to_https "true";
+  }
+     {{ '{%' }} endif {{ '%}' }}
+  {{ '{%' }} NGINX_HTTPS_REDIRECT_STRATEGY == "forward_for_proto" {{ '%}' }}
```
Comment (feanil) [inline playbooks/roles/ansible-role-django-ida/templates/templates/edx/app/nginx/sites-available/ROLE_NAME.j2]: you need an `elif` here.

**Comment by feanil** [comment]
can you generate the output of this template and post a gist of it so that we can review it?

**Comment by arbabnazar** [comment]
@e0d @feanil here is the output of this [template](https://gist.github.com/arbabnazar/41d29d30dabde9a3ea0f).

File: playbooks/roles/ansible-role-django-ida/templates/templates/edx/app/nginx/sites-available/ROLE_NAME.j2 (hunk 1 above)
Comment (arbabnazar) [inline playbooks/roles/ansible-role-django-ida/templates/templates/edx/app/nginx/sites-available/ROLE_NAME.j2]: @feanil Thanks for the tips

**Comment by e0d** [comment]
:+1:

**Comment by feanil** [comment]
:+1:

**Pull request merged**